app/core/mypayment/coredata_mypayment.py

```diff
@@ -0,0 +1,42 @@
+from fastapi import Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.core.users.models_users import CoreUser
+from app.dependencies import get_db, is_user
+from app.types.core_data import BaseCoreData
+from app.utils.tools import get_core_data
+
+
+class MyPaymentInvoiceCoordinates(BaseCoreData):
+    """Invoice coordinates for MyPayment."""
+
+    name: str = ""
+    address_street: str = ""
+    address_city: str = ""
+    address_postal_code: str = ""
+    address_country: str = ""
+    siret: str | None = None
+
+
+class MyPaymentBankAccountInformation(BaseCoreData):
+    """Bank account holder information for MyPayment."""
+
+    holder_user_id: str = ""

```

Comment:
I think you could deduct the user_id from the manager of the structure account holder